src/components/elements/card.js

```diff
@@ -109,24 +112,28 @@ export const Card = ({ children, Icon, title, content, width, min_height, is_inl
                     {is_inline_icon ? (
                         <>
                             <IconContainer>
-                                <Icon />
-                                <Header as="h4" weight="500">
-                                    {title}
-                                </Header>
+                                <div>
+                                    <Icon />
+                                </div>
+                                <CardContentContainer>
+                                    <Header as="h4" weight="bold">
```

Comment:
The default `weight` for Header component is `bold`. 
`<Header as="h4">`

Reply:
will fix it in shared component :pray: